File: Dashboard/mydash/src/app/subs/notification.ts
export class Notification {
    username: string;

    constructor(username: string) {
        this.username = username;
    }

    public getUsername(): string {
        return this.username;
    }

    public setUsername(username: string): void {
        this.username = username;
    }

    public toString(): string  {
        return "Benutzer: " + this.username;
    }
}

File: REST-Services/ipdashboard/src/main/java/de/fhaachen/ipdashboard/model/Notification.java
package de.fhaachen.ipdashboard.model;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import javax.persistence.Column;

@Entity // This tells Hibernate to make a table out of this class
@Table(name = "notification")
public class Notification {
    
    @Id
	@Column(name = "username")
	private String username;

	public String getUsername() {
		return this.username;
	}

	public void setUsername(String username) {
		this.username = username;
	}
}